nvi

```diff
@@ -10,7 +10,12 @@ N=$(tput sgr0) # Normal
 U=$(tput smul) # Underline
 
 print_usage() {
-  cat << EOF
+  less \
+    --raw-control-chars \
+    --no-init \
+    --quit-if-one-screen \
+    --QUIT-AT-EOF \
```

Comment:
Have you considered that providing complete documentation *inside CI* may not be necessary? You're only going to need it until you get things right, and then you're not going to need it again until something breaks or requirements change. The decisions here hinge on how critical you consider the current help documentation.

1. At best possible with trickery we wouldn't want to delve into.
1. You could probably embed the formatted content inside the script and redirect a file descriptor to `man`. I'm not sure I think it's a good idea. :)
1. Trivial. Sort of what we're doing now.
1. Also relatively trivial, though note that if you want `man nvi` you'll need to install the page somewhere `man` knows how to find it (`/usr/share/man/man1`?). Without installing you would have to do something like `man nvi.1` (try `man tig/doc/tig.1` for [`tig`](https://github.com/jonas/tig)). For CI you could still just install the `nvi` script.

Do you imagine installing and/or using `nvi` on a host system? If so, having a `man` page would be nice. If not, you would not want to have to download a separate file just to be able to run `man nvi.1`, though it would matter slightly less when distributed as an archive.

In any case, I propose restoring the behaviour of `cat` without using `cat`. We can keep the discussion going here but my proposed change is clearly not the right direction for `--help`.